### ISSUE TITLE
TestReports does not seem to support `verbose=true` option for `@testset` 

### ISSUE THREAD

**Issue by BambOoxX** [JuliaTesting/TestReports.jl#90]
I don't know how to provide an example for this specific problem, but `TestReports.test()` fails if I use `@testset verbose = true "Something" begin
`
Any ideas why ?

**Comment by oxinabox** [comment]
Not sure, but TestReports was developed before the `verbose=true` flag was added to `@testset`

**Comment by mmiller-max** [comment]
Thanks for raising, this should be fixed in #92 

The issue came from the fact `TestReports` wraps a packages tests in a `ReportingTestSet`, and this testset type is inherited by child test sets as per this example from the [docs](https://docs.julialang.org/en/v1/stdlib/Test/#Creating-Custom-AbstractTestSet-Types)

```Julia
@testset CustomTestSet foo=4 "custom testset inner 2" begin
    # this testset should inherit the type, but not the argument.
    @testset "custom testset inner" begin
        @test true
    end
end
```
So updating `ReportingTestSet` to handle the kwargs allowed by a `DefaultTestSet` gets it working.
